.github/workflows/main.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           cd ${{ env.VCPKG_ROOT }}
           ./bootstrap-vcpkg.sh -disableMetrics
           nuget=$(./vcpkg.exe fetch nuget | tail -n 1)
-          owner="${GITHUB_REPOSITORY%/*}"
+          owner="rfomin"
```

Comment:
@fabiangreffrath Oddly enough, NuGet packages don't work on your account. I have successfully built it in my repository: https://github.com/rfomin?page=1&tab=packages Please take a look at your settings, maybe disable uploading packages? Not sure what's wrong.

Reply:
Me neither, I have no idea what's wrong. 🤷

Reply:
There is some blubber on this page 
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry